paymentsheet/src/main/java/com/stripe/android/paymentelement/embedded/FormActivity.kt

```diff
@@ -39,7 +39,7 @@ internal class FormActivity : AppCompatActivity() {
                         finish()
                     }
                 ) {
-                    args?.selectedPaymentMethodCode?.let { Text(it) }
+                    Text((args as FormContract.Args).selectedPaymentMethodCode)
```

Comment:
You need to set this as a local variable before checking if it's null, then you won't need the cast.